queries/buildings.jinja2

```diff
@@ -57,10 +58,11 @@ SELECT
     NULL AS area,
     osm_id AS __id__,
     NULL AS "building:part",
+    "address" AS kind,
```

Comment:
More readable... could actually add up to a decent amount in a dense
building tile... but will also compress well.

On Wed, Sep 30, 2015 at 12:02 PM, Nathaniel V. KELSO <
notifications@github.com> wrote:

> In queries/buildings.jinja2
> <https://github.com/mapzen/vector-datasource/pull/246#discussion_r40813301>
> :
>
> > @@ -57,10 +58,11 @@ SELECT
> >      NULL AS area,
> >      osm_id AS __id__,
> >      NULL AS "building:part",
> > +    "address" AS kind,
>
> Using address instead of addr because the extra characters aren't that
> precious.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/vector-datasource/pull/246/files#r40813301>.
>